documentation/plain_rules.md

```diff
@@ -190,6 +191,50 @@ First time an update action using trust token is triggered, Perseo interacts wit
 
 It could happen (in theory) that a just got auth token also produce a 401 Not authorized, however this would be an abnormal situation: Perseo logs the problem with the update but doesn't try to get a new one from Keystone. Next time Perseo triggers the action, the process may repeat, i.e. first update attemp fails with 401, Perseo requests a fresh auth token to Keystone, the second update attemp fails with 401, Perseo logs the problem and doesn't retry again.
 
+NGSIv2 example:
+```json
+"action":{
+        "type":"update",
+        "parameters":{
+            "id":"${id}_mirror",
+            "version": 2,
+            "attributes": [
+                {
+                    "name":"abnormal",
+                    "type":"number",
+                    "value": 7
+                }
+            ]
+        }
+    }
+```
+
+**Note:** NGSIv2 update actions ignore the trust token for now.
+
+When using NGSIv2 in the update actions, the value field perform [string substitution](#string-substitution-syntax). If `value` is a String, Perseo will parse the value taking into account the `type` field, this only applies to *`Number`*, *`Boolean`* and *`None`* types. 
```

Comment:
This is a bit obsure... What do you think about adding an example to show how it works?

Reply:
Yes. I have reviewed the test cases and I have realized that it was necessary to parse the DateTimes correctly in order to directly use rules variables without using cast. I added the case in this commit 5ccccb7. An more tests for all possible types in 2c7902c.

New documentation for NGSIv2 types in this commit 3d148d6.